src/Couchbase.Lite.Tests.Shared/QueryTest.cs

```diff
@@ -1577,8 +1577,6 @@ public void TestLiveQueryBlocksClose()
                 }
 
                 doc1Listener.WaitForResult(TimeSpan.FromSeconds(20));
-                otherDb.Invoking(d => d.Dispose())

```

Comment:
There is no reason to keep this test around as it tests behavior that is now incorrect.  Keeping it around will only confuse people who look at it later because the name suggests that live query *should* be blocking the close when actually the test no longer tests that with this line removed.